lib/face_control/checkers/comments.rb

```diff
@@ -0,0 +1,29 @@
+require 'face_control/comment'
+
+module FaceControl
+  module Checkers
+    class Comments
+
+      def command(filenames)
+        "grep -inEH '(todo|fixme)' #{filenames}"
+      end
+
+      def parse(command_output)
+        comments = []
+        unless command_output.empty?
+          command_output.each_line do |line|
```

Comment:
Perhaps if you use the [`lines`](http://ruby-doc.org/core-2.3.1/String.html#method-i-lines) method you will be able to `map` them straight to `Comment`s and get rid of the local `comments` variable.